### PR TITLE
Install plugin User Interface

### PR DESCRIPTION
**Pull request by sjrd218**
UI for plugin upload

### CODE REVIEW

**Comment by sjrd218** [comment]
Roger that. I'll look into it, then get your input before proceeding.

**Comment by gschueler** [comment]
ideas for access control @ahonor ?

**Comment by ahonor** [comment]
Unless we overload resource=system, action=admin, we should introduce a new resource type. Eg, resource=plugin action={list,add,remove}

File: rundeckapp/grails-app/i18n/messages.properties [hunk 1]
```diff
@@ -1667,6 +1667,7 @@ passwordUtility.output.label=Output
 passwordUtility.encoder.label=Encoder:
 
 execution.html.waiting=Waiting for Execution log...
+<<<<<<< HEAD
```
Comment (gschueler) [inline rundeckapp/grails-app/i18n/messages.properties]: merge remenant

File: rundeckapp/grails-app/i18n/messages.properties [hunk 2]
```diff
@@ -1683,3 +1684,15 @@ page.users.updated.label=Updated
 page.users.lastjob.label=Last Job execution
 page.users.tokens.label=N\u00BA Tokens
 page.users.tokens.help=You can administrate the tokens in the User Profile page.
+=======
```
Comment (gschueler) [inline rundeckapp/grails-app/i18n/messages.properties]:
```suggestion
```

File: rundeckapp/grails-app/i18n/messages.properties (hunk 1 above)
Comment (gschueler) [inline rundeckapp/grails-app/i18n/messages.properties]:
```suggestion
```

File: rundeckapp/grails-app/i18n/messages.properties [hunk 3]
```diff
@@ -1683,3 +1684,15 @@ page.users.updated.label=Updated
...
+gui.admin.GetPlugins=Get Plugins
+gui.admin.InstallPlugin=Install Plugin
+plugin.form.pluginFile.label=Upload Plugin File(jar/zip)
+plugin.form.pluginUrl.label=URL to plugin file:
+plugin.install.success=Plugin Installed Successfully
+plugin.error.missing.upload.file=A plugin file must be specified
+plugin.error.missing.url=The plugin URL is required
+plugin.error.invalid.url=The plugin URL must contain a valid URL target specifying the location of the plugin file
+plugin.error.invalid.plugin=The plugin file was invalid. Please check the server logs for details.
+>>>>>>> Add a UI to the plugin page that allows a user to install a plugin via
```
Comment (gschueler) [inline rundeckapp/grails-app/i18n/messages.properties]:
```suggestion
```

**Pull request merged**